src/charm.py

```diff
@@ -1085,10 +1181,10 @@ def _scrape_jobs(self) -> list:
 
 
 def _get_loadbalancer_status(namespace: str, service_name: str):
-    client = Client()
+    client = Client()  # type: ignore
     traefik_service = client.get(Service, name=service_name, namespace=namespace)
 
-    if status := traefik_service.status:
+    if status := traefik_service.status:  # type: ignore
```

Comment:
I'm curious why, from one PR to another, a few hours apart, you suddenly need all the `type: ignore`s.